### DISCUSSION TITLE
future projections means?

### DISCUSSION THREAD

**Discussion by francescamogavero** [CLIMADA-project/climada_python#1046] [General]
This is not a problem but more of a technical information request. I am using CLIMADA.exe and when I select “future projections” and as the period 2020 (corresponding to 2010-2030) I would like to understand how the risk is calculated. Specifically when the app gives me the report and gives me a value as a return period, I would like to understand that value based on what year it was calculated. For example for the return period of 200 years for italy for the future scenario 2010-2030 I have a loss of 10,000,000 (made up), is this based on an average of the risk from 2010 to 2030?

T

**Comment by chahank** [comment]
Dear @francescamogavero ,

for questions regarding the CLIMADA app by EIOPA please refer to them directly. 

As for your question, I have a hard time answering it being quite vague. I can recommend you our documentation and related papers explaining how risk is calculated. For details about what is calculated in the CLIMADA app, please refer to the makers of the app.

Documentation: https://climada-python.readthedocs.io/en/stable/
Main risk paper: https://gmd.copernicus.org/articles/14/351/2021/
Further papers: https://www.zotero.org/groups/2502787/climada_open/collections/WZN2U7EK

General website: https://climada.ethz.ch

**Comment by francescamogavero** [comment]
@chahank @davidnbresch  
I'm using CLIMADA for probabilistic flood risk assessment with synthetic event sets, and I noticed that multiple events are simulated for each year.

In reality, extreme flood events (such as 1-in-100 or 1-in-500 year events) are very unlikely to occur in consecutive years.

Does CLIMADA model this kind of temporal dependence between events?

Or are the events treated as independent in terms of their occurrence across the years?

I'm especially interested in how this might affect multi-year risk projections or adaptation planning.

**Comment by chahank** [comment]
CLIMADA does not model hazards, but it is a framework for computing risks.  Modelling the hazard (or finding adequate datasets) is up to the user. Any temporal dependencies are possible. 

For more information on modelling multi-hazard impacts please see: https://www.nature.com/articles/s41598-024-55775-2

**Comment by francescamogavero** [comment]
I quote part of this article "When working with GCMs, each year y has a realisation driven by each GCM, resulting in the annual impact." does this mean that in a year at least one event occurs for each simulation resulting from the concatenation of these models (GCM,GHM) in italy for example? The risk modeling has been constructed in such a way that it is possible that no event happens in the simulations (always taking italy as a reference). This is because I noticed from the maps that every year there is always more than one event in Italy.

"CLIMADA does not model hazards, but it is a framework for calculating hazards. Modeling the hazard (or finding appropriate datasets) is at the user's discretion. Possible time dependencies are possible." 

does this mean, that for the time being, CLIMADA does not take into account this inter-annual spatial-temporal dependence, i.e., it does not “remember” that an event has affected a certain area in the past? (I am using CLIMADA.exe, but I think it applies anyway). I thank you for your continued helpfulness and patience, you have been very helpful. If you can answer this question for me, you will really be very helpful for my thesis in which your continued help will definitely be mentioned. :)

**Comment by chahank** [comment]
Dear @francescamogavero ,

CLIMADA is a platform to compute risk based on hazard exposure and impact functions. It is fully up to the user to choose the appropriate datasets. The hazard can be probabilistic, time series, with or without correlations. This is 100% up to the user and what data they use. Hence, inter-annual spatial-temporal dependence are taken into account as much as the user puts it in or not. 

Regarding CLIMADA.exe, please contact EIOPA directly. If I remember correctly, they use probabilistic hazard sets without time ordering. 

I can really recommend that you read carefully the provided documentation, papers, and examples and get familiar with the Python code directly.

**Comment by francescamogavero** [comment]
thank you for the answers! really very helpful. I am an actuary, so I had difficulty using CLIMADA in python as I don't know how to program. For that I relied on CLIMADA.exe provided by EIOPA. However on the time dependence I wrote to them to ask for confirmation of this, if indeed in the datasets used there is no time dependence, in this case the risk might be overestimated (which I already suspected given the very high loss values). From what I understand, if EIOPA does not provide an answer, the final dataset is provided by CaMa - flood, so I could ask them if temporal dependence was used or ISIMIP?

**Comment by chahank** [comment]
Yes. 

Note also that most CLIMADA functionalities require to learn and use Python. 

I would not subscribe to the sentence "if indeed in the datasets used there is no time dependence, in this case the risk might be overestimated ". I think this is based on a misunderstanding of how risk is computed. While it is true that cumulated impacts over a time period cannot be fully derived from a probabilistic set without making assumptions on the time correlations, the risk can be calculated. Please read carefully (and understand) how risk is defined and calculated: https://www.geosci-model-dev.net/12/3085/2019/

**Comment by francescamogavero** [comment]
I studied the risk and I report what I understood, please correct me where I make some mistakes or inaccuracies:

1. Calculate the loss for each event
For each hazard event (e.g. flood), it calculates

loss = exposure * intensity * vulnerability

So you have a list of total losses

2. Sort the events by loss, from the largest to the smallest

So: the most disastrous event is at the top of the list.
Each event has an annual frequency (in my dataset it is constant because each event from different simulations is comparable)

3. Build the Loss Exceedance Curve

To build it, the cumulative frequency of the events sorted by loss is calculated and then inverts that frequency into a return period.



However, in reasoning, I thought that something like this could happen: if I simulate events every year I will have more simulated events than perhaps in reality (where it is not true that a catastrophic event happens every year, or rather, at least in the past it was not so). If I have more simulated events, especially if they include extreme events with very high losses, this allows me on the one hand to sample better because I have the whole range of possible scenarios, but on the other hand the estimated loss for that return period increases precisely because it includes the large events that were simulated. 

As always I thank you for your answers, let me know if what I wrote has

**Comment by chahank** [comment]
These are all excellent questions.

However, I am sorry, this level of personalized consulting exceeds what we can provide. Please ask your direct supervisor for support on these general questions on risk modeling.

**Comment by francescamogavero** [comment]
I thank you for the compliment, I guess you are a researcher who deals more with the climate component, but really congratulations for the answers you were able to provide in this area. Sure, I will look into this further with my supervisor and we can stay in touch for new insights on risk modeling if you'd like! Thanks again

**Comment by chahank** [comment]
Thank you for the kind words! Also, I wish you the best with your research. I will now transform this thread into a discussion such that people who want can participate.
